tests/ProductCarousel.test.jsx

```diff
@@ -97,34 +136,27 @@ jest.mock('axios');
 jest.mock('../client/src/components/Related/card.css', () => () => (<div>Carousel Card Style Placeholder</div>));
 jest.mock('../client/src/components/Related/carousel.css', () => () => (<div>Carousel Style Placeholder</div>));
 jest.mock('../client/src/components/Related/modal.css', () => () => (<div>Modal Style Placeholder</div>));
-jest.mock('../client/src/components/Related/Modal.jsx', () => () => (<div>Modal Placeholder</div>));
 
 // TESTS =======================================================
-it('should load and display the selected product data',
-  () => axios.get('/products/1')
-    .then((productInfo) => expect(productInfo).toEqual(mockProductData)));
-
+// axios tests =================================================
 it('should load and display the related ids of the product',
-  () => axios.get('/products/1/related')
+  () => axios.get('/products/48432/related')
     .then((relatedIds) => expect(relatedIds).toEqual(mockRelatedIds)));
 
-it('should load and display the styles of the product',
-  () => axios.get('/products/48421/styles')
-    .then((productStyles) => expect(productStyles).toEqual(mockStyleData)));
+it('should load and display product info for related product data',
+  () => axios.get('/products/48433')
+    .then((relatedProductData) => expect(relatedProductData).toEqual(mockRelatedDataSecond)));
+
+it('should load and display product styles for related product data',
+  () => axios.get('/products/48433/styles')
+    .then((relatedProductStyles) => expect(relatedProductStyles).toEqual(mockRelatedDataThird)));
 
 it('should load and display carousel module title', async () => {
   render(
     <Provider store={store}>
       <RelatedItems />
     </Provider>,
   );
```

Comment:
you can take out async and check for loading... when your items are loading

Reply:
or you can delete it as the test is not currently testing anything

Reply:
Removed! Thanks @afong888 !